PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera/AttachCamera.rbxmx

```diff
@@ -66,8 +66,8 @@ local function CreateAttachCamera()
 
 
 			camera.Focus = CFrame.new(subjectPosition)
-			module.cframe = CFrame.new(camera.Focus.p - (zoom * newLookVector), camera.Focus.p)
-			self.LastCameraTransform = module.cframe
+			camera.CFrame = CFrame.new(camera.Focus.p - (zoom * newLookVector), camera.Focus.p)
+			self.LastCameraTransform = camera.CFrame
```

Comment:
Is this code tested yet? I don't recall Camera ever having a `CFrame` property, you might want to change it to `CoordinateFrame`